torch/nn/functional.py

```diff
@@ -3237,8 +3237,7 @@ def poisson_nll_loss(
     if size_average is not None or reduce is not None:
         reduction = _Reduction.legacy_get_string(size_average, reduce)
     if reduction != "none" and reduction != "mean" and reduction != "sum":
-        ret = input
-        raise ValueError(reduction + " is not a valid value for reduction")
+        raise ValueError(f"'{reduction}' is not a valid reduction mode. Expected 'none', 'mean', or 'sum'.")
```

Comment:
please combine this PR with the last PR (https://github.com/pytorch/pytorch/pull/171077/files)